CHANGES.md

```diff
@@ -1,5 +1,35 @@
 # Changes
 
+## edge-20.6.1
+
+This edge release is a release candidate for `stable-2.8`! It introduces several
+improvements and fixes for multicluster support.
+
+* CLI
+  * Added multicluster daisy chain checks to `linkerd check`
+  * Added list of successful gateways in multicluster checks section of `linkerd
+    check`
+* Controller
+  * Renamed `nginx-configuration` ConfigMap to `linkerd-gateway-config` (please
+    manually remove the former if upgrading from an earlier multicluster
+    install, thanks @mayankshah1607!)
+  * Renamed multicluster gateway ports to `mc-gateway` and `mc-probe`
+  * Fixed Service Profiles routes for `linkerd-prometheus`
+* Internal
```

Comment:
I don't have a good rubric for which internal changes should get listed in the release notes but none of these seem significant enough from the user's perspective to include.

Reply:
@adleong I don't usually add the Internal section, but these points all seemed particularly useful to mention for external contributors, especially about CI.

I also wanted to make sure Joakmir had a mention for all the work they have done on the scripts. I don't think it really belongs in another section. How would you feel if I just kept that bullet?

Reply:
SGTM